tests/ctst/common/common.ts

```diff
@@ -258,18 +271,14 @@ Given('a transition workflow to {string} location', async function (this: Zenko,
 });
 
 When('i restore object {string} for {int} days', async function (this: Zenko, objectName: string, days: number) {
-    const objName = objectName ||  this.getSaved<string>('objectName');
-    this.resetCommand();
-    this.addCommandParameter({ bucket: this.getSaved<string>('bucketName') });
-    this.addCommandParameter({ key: objName });
-    const versionId = this.getSaved<Map<string, string>>('createdObjects')?.get(objName);
-    if (versionId) {
-        this.addCommandParameter({ versionId });
-    }
-    this.addCommandParameter({ restoreRequest: `Days=${days}` });
-    await S3.restoreObject(this.getCommandParameters());
+    await restoreObject.call(this, days, objectName);
 });
 
+When('i adjust restore duration of object {string} for {int} days',
```

Comment:
"adjust restore" is not a separate step : this is not intrinsically an adjust, but only because of the condition of the object when this happens...

→ the steps should represent what we actually do (e.g. "issue an s3:restore command"), to make it easier to understand what goes on. I.e. we may have a "adjust restore" **_scenario_**, but that scenario would be simply "archive → restore → wait until restored → restore → check the restore duration has been updated"